packages/mobile/locales/en-US/nuxNamePin1.json

```diff
@@ -10,8 +10,8 @@
     "heading1": "Data and Privacy",
     "heading2": "Celo Assets and {{appName}} Account",
     "info": "In order to use our services, please read and accept our <0>User Agreement and Terms</0> by clicking the accept button below.",
-    "privacy": "By joining this network, you give us permission to collect anonymous information about your use of the app. Additionally, if you confirm your phone number, a hashed copy of it will be stored on the Celo Network.",
-    "goldDisclaimer": "When you create an ‘account’ with {{appName}} you are creating a digital wallet to which only you hold the keys.  No other person or entity, including cLabs, can recover your key, change or undo transactions, or recover lost funds. Be aware that CELO and cUSD are part of a new asset class and present a risk of financial loss.  Carefully consider your financial circumstances and tolerance for financial risk before purchasing either. You can only send up to {{dailyLimitCUSD}} cUSD per day through {{appName}} (the “Daily Limit”).  You must contact support@cLabs.co and provide additional, personal information to request increasing this Daily Limit. There could be a delay to conduct required compliance."
+    "privacy": "By joining this network, you give us permission to collect anonymous information about your use of the app. Additionally, if you confirm your phone number, a hashed copy of it will be stored on the Celo Network. If you grant Valora access to your contact list, Valora will import contacts name, phone number and profile picture as described in our <0>Privacy Policy</0>.",
```

Comment:
I'm tripping on the grammar here.  should this is possessive (`contacts'`)?  do we need an article?  How about:

` ...Valora will import each contact's name ....`

Reply:
Yeah I thought that wording was weird...will change